### DISCUSSION TITLE
how mediamtx restart, when the rtsp stream is suddenly interrupted and continues

### DISCUSSION THREAD

**Discussion by tuteming** [bluenviron/mediamtx#3494] [Questions]
### Question

After mediamtx starts, it starts pushing rtsp streaming. If the rtsp stream is suddenly interrupted and continues to be pushed after an indefinite period of time, how can mediamtx continue to operate and maintain the push?

thamks

**Comment by aler9** [comment]
Hello, the feature you are looking for is being developed and is being tracked here: #2214

[a comment by an automated account is not shown]
